config.json

```diff
@@ -1347,7 +1369,12 @@
         "name": "Connect",
         "uuid": "2fa2c262-77ae-409b-bfd8-1d643faae772",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "array-analysis",
+          "array-transformations",
+          "functions",
+          "for-loops"
```

Comment:
* classes and conditionals are missing
* I would recommend array-loops instead of for loops here because the iteration is over elements of the array
* arrays would probably also be a good fit here